ios/RCTCameraManager.h

```diff
@@ -72,11 +71,11 @@ typedef NS_ENUM(NSInteger, RCTCameraTorchMode) {
 @property (nonatomic, assign) NSInteger videoTarget;
 @property (nonatomic, assign) NSInteger orientation;
 @property (nonatomic, assign) BOOL mirrorImage;
+@property (nonatomic, assign) BOOL cropToPreview;
 @property (nonatomic, strong) NSArray* barCodeTypes;
 @property (nonatomic, strong) RCTPromiseResolveBlock videoResolve;
 @property (nonatomic, strong) RCTPromiseRejectBlock videoReject;
 @property (nonatomic, strong) RCTCamera *camera;
-@property (nonatomic, assign) BOOL cropToViewport;
```

Comment:
is this a breaking change ?

Reply:
Well, it is compared to the latest in master. But I don't think the version with that cropToViewport is released as a package?

We replaced the cropToViewport with cropToPreview and also implemented it so it works for all orientations for both android and iOS.

The old implementation was merged into master in this commit:
https://github.com/react-native-community/react-native-camera/commit/39f19cd7c7b39689f32af7ad08a26e248f0742f9

@sibelius

Reply:
well, it is time for a new release

@lwansbrough can we help us with that?

@mckn fix the conflicts and let's merge this, we can test more on master